packages/react-components/package.json

```diff
@@ -7,16 +7,17 @@
         "prepublish": "npm run clean && tsc",
         "deploy-storybook": "storybook-to-ghpages",
         "snapshot": "NODE_ENV=PERCY build-storybook && percy storybook ./storybook-static",
-        "cy:run": "npx cypress run --component",
+        "cy:run": "npx percy exec -- npx cypress run --component -b chrome",
```

Comment:
@pshoniuk any reason why we use `npx` here instead of adding `percy` and `cypress` as dev dependencies?

Reply:
There was no particular reason for it. I just did it routinely. I removed it for consistency with the existing code base.